src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/PriceRepository.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Data.Infrastructure;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Persistence;
+using NHSD.BuyingCatalogue.Solutions.Persistence.Models;
+
+namespace NHSD.BuyingCatalogue.Solutions.Persistence.Repositories
+{
+    public sealed class PriceRepository : IPriceRepository
+    {
+        private readonly IDbConnector _dbConnector;
+
+        private const string ListCataloguePricesSql = @"
+        SELECT  cp.CatalogueItemId,
+                ci.[Name] as CatalogueItemName,
+                cp.CataloguePriceId,
+                cp.CataloguePriceTypeId,
+                pu.PricingUnitId,
+                pu.[Name] as PricingUnitName,
+                pu.[Description] as PricingUnitDescription,
+                pu.TierName as PricingUnitTierName,
```

Comment:
Can you make the `AS` syntax upper case (I think it reads better and it keeps it consistent).

Reply:
👍